src/OrchardCore.Modules/OrchardCore.Email.Smtp/Services/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Email.Services;
+using OrchardCore.Email.Smtp.Services;
+
+namespace OrchardCore.Email.Azure.Services;
+
+public static class ServiceCollectionExtensions
+{
+    public static IServiceCollection AddSmtpEmailProvider(this IServiceCollection services)
+    {
+#pragma warning disable CS0612 // Type or member is obsolete
+        services.AddSmtpService();
+#pragma warning restore CS0612 // Type or member is obsolete
+
+        services.AddEmailProviderOptionsConfiguration<SmtpProviderOptionsConfigurations>();
+
+        return services;
+    }
+
+    [Obsolete]
```

Comment:
Add a message.

Reply:
No need. It’s a decorator for a private method. We can just remove it when we remove the `ISmtpService`